tensorflow/python/platform/self_check.py

```diff
@@ -61,4 +61,3 @@ def preload_check():
     # incompatibilities before we trigger them (which would typically result in
     # SIGILL).
     from tensorflow.python.platform import _pywrap_cpu_feature_guard
-    _pywrap_cpu_feature_guard.InfoAboutUnusedCPUFeatures()
```

Comment:
You can add this back now.  We fixed the double-printing with [0e1b2b4](https://github.com/tensorflow/tensorflow/commit/0e1b2b44c706afade2b488aecc71f36940165e74).

Reply:
@cantonios I added it back.